edgexfoundry/device-sdk-go
# refactor!: Rework code for refactored MessageBus Configuration

BREAKING CHANGE: MessageQueue renamed to MessageBus and fields changed. See v3 Migration guide.

This PR dependent on https://github.com/edgexfoundry/go-mod-bootstrap/pull/410

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-sdk-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  TBD

## Testing Instructions
Run non-secure Edgex Stack with device virtual and stop all core/support services.
Build example locally
Run example locally with DEBUG log level
Verify example bootstraps properly and connects to the MessageBus
Verify example pushing events events.

## New Dependency Instructions (If applicable)
